ui/store/actions.ts

```diff
@@ -4759,7 +4760,7 @@ export function signAndSendSmartTransaction({
     );
     const signedCanceledTransactions = await createSignedTransactions(
       unsignedTransaction,
-      smartTransactionFees.cancelFees,
+      [],
```

Comment:
Hey, it looks like this change might cause signedCanceledTransactions to always be empty. Could you take a look and confirm?

Reply:
I've removed `const signedCanceledTransactions`, since it's no longer needed. The STX controller still expects the `signedCanceledTransactions` param, so I'm passing an empty array there now until it's deprecated in the STX controller.